dustpress.php

```diff
@@ -655,6 +655,76 @@ private function populate_data_collection() {
 		return apply_filters( 'dustpress/data/wp', $wp_data );
 	}
 
+	/**
+	*  Gets permalink.
+	*
+	*  @type	function
+	*  @date	11/1/2019
+	*  @since	1.24.2
+	*
+	*  @return	string Permalink.
+	*/
+	public function dp_get_permalink() {
+
+		// Get request uri safely.
+		$request_uri = filter_var( $_SERVER['REQUEST_URI'], FILTER_SANITIZE_URL );
+		$permalink 	 = $this->dp_get_home_url() . $request_uri;
```

Comment:
Wouldn't this create problems in installs that are not done on the root of a domain? If the home url was `https://www.domain.com/subfolder` and `$request_uri` was `/subfolder/some-page` this method would return `https://www.domain.com/subfolder/subfolder/some-page`.